SCF-Assignment-7/SparseMatrix.java

```diff
@@ -0,0 +1,222 @@
+package SparseMatrix;
+
+/**
+ * 
+ * @author Arjita
+ *
+ */
+public class SparseMatrix {
+	private int[][] sparseMatrix;// 2D array to store non-zero elements of
+									// sparse matrix
+	private int numberOfRows;// number of rows of input array
+	private int numberOfCol;// number of columns of input array
+
+	/**
+	 * SparseMatrix is constructor of class which will initialise the sparse
+	 * matrix array in which there will be two columns, first column will store
+	 * value and second column will store the index This will decrease the space
+	 * complexity.
+	 * 
+	 * @param inputArray
+	 *            input sparse matrix
+	 */
+	public SparseMatrix(int[][] inputArray) {
+		numberOfRows = inputArray.length;// number of rows of array
+		numberOfCol = inputArray[0].length;// number of column of array
+		int countElements = 0;// count the non zero elements of input matrix
+		for (int i = 0; i < inputArray.length; i++) {
+			for (int j = 0; j < inputArray[i].length; j++) {
+				if (inputArray[i][j] != 0) {
+					countElements++;
+				}
+			}
+		}
+		sparseMatrix = new int[countElements][2];// first column will store

```

Comment:
Add the inline comment above the line, so that we do not have a blank line just like this for just comment